cmd/kola/bootchart.go

```diff
@@ -37,6 +39,8 @@ init process.
 This must run as root!
 `}
 
+var kolaPlatform = flag.String("platform", "qemu", "compute platform for bootchart")
```

Comment:
Sounds good. (Bootchart is just happens to be the only command that uses the flag anymore)